service/internal/integrationtest/s3/endpoint_url_test.go

```diff
@@ -0,0 +1,187 @@
+package main
```

Comment:
this package being main is tripping the integ tests

> found packages s3 (api_test.go) and main (endpoint_url_test.go) in /codebuild/output/src1559507751/src/github.com/aws/aws-sdk-go-v2/service/internal/integrationtest/s3

Reply:
I also 100% did not apply the integration test flag so that needs fixed.

Reply:
fixed